app/jobs/feed_entry_update_job.rb

```diff
@@ -60,9 +83,13 @@ def receive_feed_entry_delete(data)
     podcast.try(:publish!)
   end
 
+  def update_sent
+    @_update_sent ||= Time.parse(message[:sent_at])
```

Comment:
You are right, I am being paranoid, it should be set, but just in case the `announce` gem changes or has some bug I don't expect, I'm making sure things will continue as gracefully as possible.